docs/restrict-cross-namespace-associations.asciidoc

```diff
@@ -0,0 +1,97 @@
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/restrict-cross-namespace-associations.html[View this document on the Elastic website]
+****
+endif::[]
+[id="{p}-restrict-cross-namespace-associations"]
+=== Restrict cross namespace resource associations
+
+By default, associations can be created between any resources as long as they are deployed in some namespaces managed by the same ECK instance.
```

Comment:
```suggestion
By default, associations can be created between any resources as long as they are deployed in namespaces managed by the same ECK instance.
```

I also wonder if it's clearer to just say "managed by the same instance of the ECK operator". This is hard to explain simply :/

Reply:
Can we turn into active voice?
```suggestion
By default, you can create associations between any resources as long as they are deployed in some namespaces managed by the same ECK instance.
```